src/import/import-generator.c

```diff
@@ -230,8 +230,12 @@ static int transfer_generate(const Transfer *t, size_t c) {
 
         assert(t);
 
+        _cleanup_free_ char *e = unit_name_escape(t->local);
```

Comment:
Yeah, `t->local` must be generated if unspecified.
Please add `assert(t->local)` in the above.